include/capstone/hppa.h

```diff
@@ -0,0 +1,538 @@
+    HPPA_INS_MTSM,
+    HPPA_INS_MTSP,
+    HPPA_INS_NOP,
+    HPPA_INS_OR,
+    HPPA_INS_PDC,
+    HPPA_INS_PDTLB,
+    HPPA_INS_PDTLBE,
+    HPPA_INS_PERMH,
+    HPPA_INS_PITLB,
+    HPPA_INS_PITLBE,
+    HPPA_INS_PMDIS,
+    HPPA_INS_PMENB,
+    HPPA_INS_POPBTS,
+    HPPA_INS_PROBE,
+    HPPA_INS_PROBEI,
+    HPPA_INS_PROBER,
+    HPPA_INS_PROBERI,
+    HPPA_INS_PROBEW,
+    HPPA_INS_PROBEWI,
+    HPPA_INS_PUSHBTS,
+    HPPA_INS_PUSHNOM,
+    HPPA_INS_RET,
+    HPPA_INS_RFI,
+    HPPA_INS_RFIR,
+    HPPA_INS_RSM,
+    HPPA_INS_SHDWGR,
+    HPPA_INS_SHLADD,
+    HPPA_INS_SH1ADD,
+    HPPA_INS_SH1ADDL,
+    HPPA_INS_SH1ADDO,
+    HPPA_INS_SH2ADD,
+    HPPA_INS_SH2ADDL,
+    HPPA_INS_SH2ADDO,
+    HPPA_INS_SH3ADD,
+    HPPA_INS_SH3ADDL,
+    HPPA_INS_SH3ADDO,
+    HPPA_INS_SHD,
+    HPPA_INS_SHRPD,
+    HPPA_INS_SHRPW,
+    HPPA_INS_SPOP0,
+    HPPA_INS_SPOP1,
+    HPPA_INS_SPOP2,
+    HPPA_INS_SPOP3,
+    HPPA_INS_SSM,
+    HPPA_INS_STB,
+    HPPA_INS_STBS,
+    HPPA_INS_STBY,
+    HPPA_INS_STBYS,
+    HPPA_INS_STD,
+    HPPA_INS_STDA,
+    HPPA_INS_STDBY,
+    HPPA_INS_STH,
+    HPPA_INS_STHS,
+    HPPA_INS_STW,
+    HPPA_INS_STWA,
+    HPPA_INS_STWAS,
+    HPPA_INS_STWS,
+    HPPA_INS_STWM,
+    HPPA_INS_SUB,
+    HPPA_INS_SUBB,
+    HPPA_INS_SUBBO,
+    HPPA_INS_SUBI,
+    HPPA_INS_SUBIO,
+    HPPA_INS_SUBO,
+    HPPA_INS_SUBT,
+    HPPA_INS_SUBTO,
+    HPPA_INS_SYNC,
+    HPPA_INS_SYNCDMA,
+    HPPA_INS_TOCDIS,
+    HPPA_INS_TOCEN,
+    HPPA_INS_UADDCM,
+    HPPA_INS_UADDCMT,
+    HPPA_INS_UXOR,
+    HPPA_INS_VDEP,
+    HPPA_INS_VDEPI,
+    HPPA_INS_VEXTRS,
+    HPPA_INS_VEXTRU,
+    HPPA_INS_VSHD,
+    HPPA_INS_XMPYU,
+    HPPA_INS_XOR,
+    HPPA_INS_ZDEP,
+    HPPA_INS_ZDEPI,
+    HPPA_INS_ZVDEP,
+    HPPA_INS_ZVDEPI,
+    
+    HPPA_INS_ENDING
+} hppa_insn;
+
+
+/// HPPA space select operand
+typedef struct hppa_mem {
+    unsigned int base;
+    unsigned int space;
+    enum cs_ac_type base_access;
+} hppa_mem;
+
+// Instruction operand
+typedef struct cs_hppa_op {
+	enum hppa_op_type type; // operand type
+	union {
+		unsigned int reg; // register value for REG operand
+		int64_t imm;      // immediate value for IMM operand
+        struct hppa_mem mem;
+	};
+	enum cs_ac_type access;
+} cs_hppa_op;
+
+// Instruction structure
+typedef struct cs_hppa {
+	// Number of operands of this instruction,
+	// or 0 when instruction has no operand.
+	uint8_t op_count;
+	cs_hppa_op operands[5]; // operands for this instruction.
+} cs_hppa;
+
+/// hppa string/integer modifier
+typedef struct hppa_modifier {
+    int type;
+    union
+    {
+        char str_mod[8];
+        uint32_t int_mod;
+    };
+    
+} hppa_modifier;
+
+// Additional instruction info
+typedef struct hppa_ext {
+    struct hppa_modifier modifiers[5];
+    uint8_t mod_num;
+    bool b_writeble;
+    bool is_alternative;
+} hppa_ext;
+
+
+
+
+// Group of HPPA instructions
+typedef enum hppa_insn_group {
+	HPPA_GRP_INVALID = 0, ///< = CS_GRP_INVALID
+
+	HPPA_GRP_COMPUTATION,
```

Comment:
```suggestion
	HPPA_GRP_COMPUTATION = 128,
```

Otherwise, it shadows the other `cs_group_type` like `CS_GRP_JUMP`.